doc/chips/11.rst

```diff
@@ -22,11 +24,8 @@ we considered a variety of features. Some of them were accepted
 and some of them were rejected. This document records some of
 the discussed-and-rejected features for posterity.
 
-Description
------------
-
 Alternative syntax for distinguishing Phase 1 and Phase 2 in an initializer
-***************************************************************************
+---------------------------------------------------------------------------
```

Comment:
I'm guessing this change was made for the sake of the outline?  I would have a way to distinguish the rationale and various rejected decisions - either a common header or prefix for the latter would do.